### DISCUSSION TITLE
Toggle Alpha

### DISCUSSION THREAD

**Discussion by iberck** [pubpub-zz/ppInk#235] [Ideas]
Similar to toggle Fading:

It would be very useful for my workflow to have "Toggle Alpha": a key combination that applies an alpha level to the current pen color (the alpha level would have to be predefined in the settings).

**Comment by pubpub-zz** [comment]
most of the time, changing alpha means also changing width. because of that the best is to assign a thin opaque definition to a first pen, and assign a large transparent pen to another one

**Comment by iberck** [comment]
This is very valuable that you comment, I did not know it, thanks for sharing it.

I currently use 9 solid colors and their alpha variants, so it's awkward to manually change the alpha level of the pencil every time I need it.
I am attaching a video of how demohelper currently does it, which pressing the T (transparent) key changes the pencil from solid to alpha and vice versa (it is very useful):

https://user-images.githubusercontent.com/182590/207481743-974440d5-c2c6-4158-ab63-13b2d46b5945.mp4
